src/box/lua/load_cfg.lua

```diff
@@ -103,6 +103,7 @@ local default_cfg = {
     replication_connect_quorum = nil, -- connect all
     replication_skip_conflict = false,
     replication_anon      = false,
+    replication_num_threads = 1,
```

Comment:
Let's call the new option simply 'replication_threads' - this would be consistent with 'iproto_threads'.

Reply:
Sure